core/commands/generate_product_repositories_command.rb

```diff
@@ -780,22 +780,34 @@ def create_repository(product)
     info_and_log("Generating repository configuration for #{product}")
     begin
       releases = send("parse_#{product}".to_sym, @config[product])
-    rescue StandardError => error
-      error_and_log("#{product} was not generated. Try again.")
-      error_and_log_error(error)
-      return false
+      if releases
+        write_product(product, releases)
+        [false, product]
+      else
+        error_and_log("#{product} was not generated. Skiped.")
+        [nil, product]
+      end
+    rescue StandardError => e
+      error_and_log("Error message: #{e.message}")
+      error_and_log("#{product} was not generated.")
+      [true, product]
     end
-    write_product(product, releases)
-    true
   end
 
   # Print summary information about the created products
   # @param products_with_errors [Array<String>] product names that were not generated
-  def print_summary(products_with_errors)
+  def print_summary(products)
     info_and_log("\n--------\nSUMMARY:\n")
-    @products.sort.each do |product|
-      result = products_with_errors.include?(product) ? '-' : '+'
-      info_and_log("  #{product}: #{result}")
+    products.each do |product|
+      case (product[0])
+      when false
+        result = '+'
+      when nil
```

Comment:
I do not see a reason for us to print this information. If the product was not configured, then it should not present in the summary table.